### PR TITLE
Fix broken resolve for kebab case addons

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: src/test/fixtures/installer-versions.ts [hunk 1]
```diff
@@ -1,450 +1,165 @@
...
-    "1.0.4-14.2.21",
-  ],
-  contour: ["1.22.0", "1.21.1", "1.21.0", "1.20.1", "1.20.0", "1.19.1", "1.18.0", "1.16.0", "1.15.1", "1.14.1", "1.14.0", "1.13.1", "1.13.0", "1.12.0", "1.11.0", "1.10.1", "1.7.0", "1.0.1", "0.14.0"], // cron-contour-update
-  registry: [
-    // cron-registry-update
-    "2.8.1",
-    "2.7.1",
-  ],
-  prometheus: [
-    // cron-prometheus-update
```
Comment (laverya) [inline src/test/fixtures/installer-versions.ts:132]: these comments are semantically significant - they're how new versions are added to the file

Comment (laverya) [reply src/test/fixtures/installer-versions.ts:132]: so we can't get rid of them

Comment (laverya) [reply src/test/fixtures/installer-versions.ts:132]: actually, that raises the question - did we break automatically adding new addon versions with this change?

Comment (laverya) [reply src/test/fixtures/installer-versions.ts:132]: Or is this just used in tests?

Comment (emosbaugh) [reply src/test/fixtures/installer-versions.ts:132]: This is just test fixtures. The real file is in the Kurl repo

Comment (laverya) [reply src/test/fixtures/installer-versions.ts:132]: 🤦

File: src/installers/installer-versions.ts [hunk 2]
```diff
@@ -51,6 +51,11 @@ async function getInternalAddonVersions(distUrl: string, kurlVersion: string) {
   const installerVersions = body.supportedVersions;
   Object.keys(installerVersions).map((addon: string) => {
     installerVersions[addon] = installerVersions[addon].filter((version: string) => version !== "latest");
+    // converts kebab case add-ons to camel case
+    // e.g. cert-manager => certManager
+    if (addon.includes("-")) {
+      installerVersions[_.camelCase(addon)] = installerVersions[addon];
+    }
```
Comment (laverya) [inline src/installers/installer-versions.ts:58]: will this carry through to the actual installer? If so, will it break package resolution?

definitely something to test on staging

**Review by laverya — APPROVED** [review]

**Pull request merged**